utils/defs.tex

```diff
@@ -84,8 +84,8 @@
 \newcommand{\EnvVarDecl}[1]{\EnvVarRef{#1}}
 \newcommand{\FuncDecl}[1]{{\ListingsCurrentStyle{#1}}\FuncIndex{#1}}
 \newcommand{\FuncParam}[1]{<{\ListingsKeywordStyle{#1}}>}
-\newcommand{\LibConstDecl}[2][\CorCppFor]{%
-  \parbox[t]{0pt}{~\\[-4pt] #1: \\\hspace*{8mm} \LibConstRef{#2} \\~}}
+\newcommand{\LibConstDecl}[2][\CorCpp]{%
+  \parbox[t]{5cm}{~\\[-4pt] #1: \\\hspace*{8mm} \LibConstRef{#2} \\~}}
 \newcommand{\LibHandleDecl}[2][\CorCppFor]{%
```

Comment:
@jdinan Can you change the default for `\LibHandleDecl` to be `CorCpp` and remove [the explicit `CorCpp` in `shmem_ctx_t`](https://github.com/openshmem-org/specification/blob/bc79f9a13a00452a8d4c87683d59213797722631/content/library_handles.tex#L16)?

Reply:
Done -- 660e2af